alf/layers.py

```diff
@@ -309,6 +328,10 @@ def forward(self, inputs):
             y = inputs.matmul(self._weight.t())
             if self._use_bias:
                 y += self._bias
+        if self._use_ln:
```

Comment:
What if self._use_ln and self._use_bn are both True? Why it's the current order being applied?

Reply:
In general, they should not be applied at the same time.

Reply:
So maybe add a check for exclusion?

Reply:
But I guess if someone really want to use both, that's ok.